src/Migrations/M056UpdateQuickLinksClassName.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Update Quick Links Block with a new ClassName
+ */
+class M056UpdateQuickLinksClassName extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    public static function execute(MigrationRecord $record): void
+    {
+        $check_is_valid_block = function ($block) {
+            return self::check_is_valid_block($block);
+        };
+
+        $transform_block = function ($block) {
+            return self::transform_block($block);
+        };
+
+        Utils\Functions::execute_block_migration(
+            Utils\Constants::BLOCK_TEMPLATE_QUICK_LINKS,
+            $check_is_valid_block,
+            $transform_block,
+        );
+    }
+    // phpcs:enable SlevomatCodingStandard.Functions.UnusedParameter
+
+    /**
+     * Check whether the Quick Links blocks is into a page.
+     *
+     * @param array $block - A block data array.
+     */
+    private static function check_is_valid_block(array $block): bool
+    {
+       // Check if the block is an array, has a 'blockName' key.
+        if (!is_array($block) || !isset($block['blockName'])) {
+            return false;
+        }
+
+        // Check if the block is a Quick Links block.
+        return
+            $block['blockName'] === Utils\Constants::BLOCK_TEMPLATE_QUICK_LINKS
+        ;
+    }
+
+    /**
+     * Update class name
+     *
+     * @param array $block - A block data array.
+     * @return array - The transformed block.
+     */
+    private static function transform_block(array &$block): array // Using the parameter as a reference to update it
+    {
+        $iterate = function (array &$innerBlocks, string $blockName, array $path = []) use (&$iterate): ?array {
```

Comment:
Instead of using this `$iterate`, can't we just create a separate function like we do [here](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Migrations/M050AddTagsBackInPostsListBlock.php#L72) for example?